plugin/core/url.py

```diff
@@ -31,6 +32,7 @@ def view_to_uri(view: sublime.View) -> str:
     return filename_to_uri(file_name)
 
 
+@deprecated("Use parse_uri() instead")
```

Comment:
is there any visible benefit in using the deprecated decorator?

I expected to see a log in the Sublime Text console, 
when a function marked with this decorator is called.

Reply:
Not in the console, but for example Pyright reports a diagnostic if the deprecated functions are used (I have the severity set to "warning" here via `python.analysis.diagnosticSeverityOverrides`):

![deprecated](https://github.com/sublimelsp/LSP/assets/6579999/bf0c3822-ae3c-414e-9ec4-d76ebf1794d5)